exec/java-exec/src/main/java/org/apache/drill/exec/planner/cost/DrillRelMdDistinctRowCount.java

```diff
@@ -174,7 +174,7 @@ private Double getDistinctRowCount(DrillScanRelBase scan, RelMetadataQuery mq, D
     }
   }
 
-  public Double getDistinctRowCount(DrillJoinRelBase joinRel, RelMetadataQuery mq, ImmutableBitSet groupKey,
+  private Double getDistinctRowCount(DrillJoinRelBase joinRel, RelMetadataQuery mq, ImmutableBitSet groupKey,
```

Comment:
Wouldn't it be better to rename this (and the other private methods here) with an 'internal' suffix to avoid potential issues in the future ?  Only the Override methods remain unchanged.   What do you think ?

Reply:
Done